components/ContactUsForms/NewsAndEventsForm/NewsAndEventsForm.vue

```diff
@@ -238,22 +239,19 @@ export default {
     async sendForm() {
       const config = useRuntimeConfig()
       this.isSubmitting = true
-      const fileName = propOr('', 'name', this.file)
-      const description = `
-        <b>Contact Information</b><br><br>
-        <b>First Name:</b><br>${this.form.user.firstName}<br><br>
-        <b>Last Name:</b><br>${this.form.user.lastName}<br><br>
-        <b>E-mail:</b><br>${this.form.user.email}<br><br>
-        <b>News or Event Details:</b><br><br>
-        <b>Title:</b><br>${this.form.title}<br><br>
-        <b>Summary:</b><br>${this.form.summary}<br><br>
-        ${fileName != '' ? `<b>File Attachment:</b><br>${fileName}<br><br>` : ''}
-        <b>Supporting Information links:</b><br>${this.supportingLinksText}<br>
-        <b>Event Specific Details:</b><br><br>
-        <b>Location:</b><br>${this.locationText}<br><br>
-        <b>Start Date:</b><br>${this.form.startDate == '' ? 'N/A' : new Date(this.form.startDate).toDateString()}<br><br>
-        <b>End Date:</b><br>${this.form.endDate == '' ? 'N/A' : new Date(this.form.endDate).toDateString()}
-      `
+      const description = `Contact Information
```

Comment:
Is Sue ok with this all being sent as a description, or does she want each of the individual columns in the table to be populated?